prime-router/src/main/kotlin/Metadata.kt

```diff
@@ -96,6 +96,7 @@ class Metadata : Logging {
         IsValidCLIA(),
         InDateInterval(),
         FilterOutNegativeAntigenTestType(),
+        Intersect()
```

Comment:
You still need code to fail if someone uses the conditionFilter on a COVID 19 receiver.

Reply:
We did that in a previous ticket where it would not allow them to run reloadSettings if a COVID 19 receiver had the condition filter. Does that need to be added to the code somehow as well even though they cannot save it?

Reply:
reloadSettings is not something a normal admin does.  You need to make sure the validation code works for loading the settings, or loading new settings to the DB.

Reply:
I checked and it works when I use `./prime receiver set....` as well. Is there another place to test/check?